lib/index.js

```diff
@@ -46,6 +46,25 @@ function programError(options, message) {
   }
 }
 
+function mergeArrays(arr1, arr2) {
+  return [...new Set([...arr1 || [], ...arr2 || []])]
+}
+
+/**
+ * Merge options by a standard way, but if some properties both are arrays, then merge them also.
```

Comment:
You should specify in the JSDOC that is a shallow merge (as opposed to a recursive or deep merge).

Reply:
OK